python/cudf/cudf/core/column/string.py

```diff
@@ -4925,7 +4925,12 @@ def find_and_replace(self, to_replace, replacement, all_nan):
         replacement = column.as_column(replacement, dtype=self.dtype)
         return libcudf.replace.replace(self, to_replace, replacement)
 
-    def fillna(self, fill_value):
+    def fillna(self, fill_value, method=None):
+        if method is not None:
+            raise NotImplementedError(
+                "fillna for string column with `method` parameter is not"
+                "supported."
+            )
         if not is_scalar(fill_value):
             fill_value = column.as_column(fill_value, dtype=self.dtype)
         return libcudf.replace.replace_nulls(self, fill_value, dtype="object")
```

Comment:
Should we call `super().fillna()` here as well?

Reply:
`libcudf` don't have string column supported at this moment. But will add to it once https://github.com/rapidsai/cudf/pull/7004 merges.

Reply:
Still, doesn't `Column.fillna()` contains the call to `libcudf.replace.replace_nulls`? My suggestion is to call `super().fillna()` so that we don't replicate it here. Maybe I'm missing something

Reply:
Ah yes, sorry I misread your comment, thought it was commented on the `NotImplementedError` line. Will change the return line to using `super`.